backends/openvino/partitioner.py

```diff
@@ -0,0 +1,141 @@
+# Copyright (c) Intel Corporation
+#
+# Licensed under the BSD License (the "License"); you may not use this file
+# except in compliance with the License. See the license file in the root
+# directory of this source tree for more details.
```

Comment:
Is this falling back to ExecuTorch license or you have your own? If former just directly mention/point to it

Reply:
> Updated